### ISSUE TITLE
Node objects remain registered to Azure clusters after rolling update

### ISSUE THREAD

**Issue by rfranzke** [gardener/machine-controller-manager#54]
When doing a rolling update on an Azure cluster, the machine controller does correctly roll the VMs, however, some `Node` objects of old VMs keep registered. On other clouds like AWS, GCP, OpenStack those objects are purged by the kube-controller-manager.

**Comment by hardikdr** [comment]
Yes, we saw this behavior in k8s 1.9.3 on azure, however it works fine with older versions. With k8s 1.8.7, it worked well, all the node-objects were removed by kube-controller-manager. 
We shall follow up with the community on this.

**Comment by vlerenc** [comment]
Let's open a Kubernetes ticket and check 1.9.4. At present 1.9 is generally disabled in Gardener: https://github.wdf.sap.corp/kubernetes/landscape-setup/blob/master/components/gardener/resources/10-cloudprofile-az.yaml#L13.
